docs/documentation/server_admin/topics/identity-broker/kubernetes.adoc

```diff
@@ -0,0 +1,27 @@
+
+[[_identity_broker_kubernetes]]
+=== Kubernetes identity providers
+
+:tech_feature_name: Kubernetes
+:tech_feature_id: kubernetes
+include::../../topics/templates/techpreview.adoc[]
+
+A Kubernetes identity provider supports authenticating clients with Kubernetes service account tokens.
+
+.Procedure
+. Click *Identity Providers* in the menu.
+. From the `Add provider` list, select `Kubernetes`.
++
+. Enter your initial configuration options or proceed with the defaults.
++
+.Kubernetes settings
+|===
+|Configuration|Description
+
+|Alias
+|The alias for the identity provider is used to link a client to the provider
+
+|Kubernetes JWKS URL
+|The Kubernetes JWKS URL when accessing an external Kubernetes cluster. The JWKS endpoint must not require authentication
```

Comment:
We should state somewhere in these docs, maybe the introduction, that if the JWKS URL is not explicitly defined, then we default to `https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT_HTTPS}/openid/v1/jwks` and authentication is handled automatically using `/var/run/secrets/kubernetes.io/serviceaccount/token`.

We can then make it clear that this setting is optional.

Reply:
Addressed.